docs/l10n.rst

```diff
@@ -0,0 +1,148 @@
+Localization (l10n)
+=====================
+
+These notes are primarily to support localizers working on localizations
+related to https://addons.mozilla.org/.
+
+Projects
+--------
+
+There are two core projects for AMO that require localization:
+
+    * `addons-server <https://github.com/mozilla/addons-server/>`_ (`addons-server on Pontoon`_)
+    * `addons-frontend <https://github.com/mozilla/addons-frontend/>`_ (`addons-frontend on Pontoon`_)
+
+Both of these projects use `Pontoon`_ for translations.
+
+Addons-server
```

Comment:
This is a nitpick but I think we (should) stylise the repo with a lowercase "a".